src/Components/Components/src/Rendering/RenderBatchBuilder.cs

```diff
@@ -58,6 +66,22 @@ public RenderBatch ToBatch()
                 DisposedComponentIds.ToRange(),
                 DisposedEventHandlerIds.ToRange());
 
+        public void InvalidateParameterViews()
+        {
+            // Wrapping is fine because all that matters is whether a snapshotted value matches
+            // the current one. There's no plausible case where it wraps around and happens to
+            // increment all the way back to a previously-snapshotted value on the exact same
+            // call that's checking the value.
+            if (_parameterViewValidityStamp == int.MaxValue)
+            {
+                _parameterViewValidityStamp = int.MinValue;
+            }
+            else
+            {
+                _parameterViewValidityStamp++;
+            }
```

Comment:
In case you're wondering, I preferred to use `Int32` not `Int64` for `_parameterViewValidityStamp` because it gets copied around a lot, plus Mono WebAssembly behaves as 32-bit.

It probably makes no significant difference, but `Int32` is perfectly sufficient here, given we can wrap around like this.